servers/rendering/rendering_device_commons.cpp

```diff
@@ -711,12 +711,12 @@ uint32_t RenderingDeviceCommons::get_image_format_required_size(DataFormat p_for
 
 	uint32_t pixel_size = get_image_format_pixel_size(p_format);
 	uint32_t pixel_rshift = get_compressed_image_format_pixel_rshift(p_format);
-	uint32_t blockw, blockh;
+	uint32_t blockw = 0, blockh = 0;
```

Comment:
I would put one variable per line. It's too easy to make the mistake of thinking that `int a, b = 0;` initializes both.

Reply:
I tried not to touch that line much since it wasn't the main target of this PR, but, in retrospective, it's a bit silly that while I was at it I didn't go all the way to rewrite it following the norm across the codebase. About to push with the change.